src/components/ListItem.tsx

```diff
@@ -160,17 +188,21 @@ const ListItem = ({
       onLongPress={() => updatePressed(true)}
       pressed={pressed}
       delayLongPress={200}
+      feedback={feedback}
       testID='list-item'>
-      {iconToRender}
-      <FlexContainer>
-        {titleToRender}
-        <DescriptionContainer>
-          {badgeLabel && <BadgeLabel pressed={pressed}>{badgeLabel}</BadgeLabel>}
-          {description && description.length > 0 && <Description pressed={pressed}>{description}</Description>}
-        </DescriptionContainer>
-        {children}
-      </FlexContainer>
-      {rightChildrenToRender}
+      <FeedbackBadge levelIdentifier={levelIdentifier} setFeedback={setFeedback} />
```

Comment:
I think this breaks other ListItems, e.g. the icons in the Vocabulary list, maybe it would make sense, to have a prop if feedback should be displayed, because currently it is always displayed ig a levelIdentifier is passed, or?

![lun-407](https://user-images.githubusercontent.com/18463686/191499306-14b4cca8-7755-4ac5-bf14-bdbf299dd273.png)

Reply:
Maybe it would be better to have Something like a ListItemWithFeedback or something, where the List Item is used inside?

Reply:
> Maybe it would be better to have Something like a ListItemWithFeedback or something, where the List Item is used inside?

Good idea. Will do.

Reply:
@SteffiStuffel I just had another look at this, the misalignment wasn't from the badge itself, but rather from a `align-items: center` that got lost somewhere along the way. Should be fixed now.

The `levelIdentifier` also only gets passed from `ExerciseScreen` and not passing it means `FeedbackBadge` returns `null`, so your first suggestion is already implemented.

Please have another look at if the current state is satisfactory.

(A separate component *would* be nice, I agree, but wouldn't really reduce the complexity in the `ListItem` component, since it still needs to react to state updates from the badge (mainly setting the border styles))